src/components/ContentSidebar/SidebarContent.js

```diff
@@ -14,7 +14,7 @@ type Props = {
 };
 
 const SidebarContent = ({ actions, title, children }: Props) => (
-    <div className="bcs-content">
+    <div className="bcs-content" data-testid="bcs-content">
```

Comment:
It seems like we will have to add a bit of these to our code. Can we remove them with something like https://www.npmjs.com/package/babel-plugin-react-remove-properties ?

Reply:
We can look into doing so in a follow-up PR. I'd like to avoid transforming the lib/dist output with this change, if possible.